sixtyfps_compiler/fileaccess.rs

```diff
@@ -44,6 +48,20 @@ mod builtin_library {
 
     use super::VirtualFile;
 
+    pub(crate) fn styles() -> Vec<&'static str> {
```

Comment:
I suggest to return an `impl Iterator<Item = &'static str>` and skip the `collect()` below. After all at the call site there's no need to really collect this into a vector.

Reply:
Why? This is way simpler and not in any critical path whatsoever.

Reply:
I think returning an iterator is idiomatic Rust.